src/singletons/Settings.hpp

```diff
@@ -526,6 +527,9 @@ class Settings
         "/notifications/suppressInitialLive", false};
 
     BoolSetting notificationToast = {"/notifications/enableToast", false};
+    BoolSetting createShortcutForToasts = {
+        "/notifications/createShortcutForToasts",
+        Modes::instance().isPortable ? false : true};
```

Comment:
add a , at the end of the ternary, that way it formats a bit nicer, causing nicer diffs in the future if we have to edit it